server/routes/pokemons/index.ts

```diff
@@ -0,0 +1,33 @@
+import express from 'express';
+import pokemonController from '../../controllers/pokemons/index.js';
+
+const routes  = express.Router();
+
+routes.route('/:name')
+  .get(async (req, res) => {
+    const name = req.params.name;
+    const pokemon = await pokemonController.getPokemonByName(name);    
+    res.end(JSON.stringify(pokemon));
+  });
+
+routes.route('/')
+.get(async (req, res) => {
```

Comment:
Please use prettier to fix all the unnecessary/missing spaces

Reply:
added